vnext/src/RNTester/js/RNTesterList.windows.ts

```diff
@@ -149,7 +149,7 @@ const APIExamples: Array<IRNTesterExample> = [
   },
   {
     key: 'AlertExample',
-    module: require('react-native/RNTester/js/AlertExample').AlertExample,
+    module: require('./AlertExample.windows'),
```

Comment:
Nit: I don't think you need the ".windows" here. We should be able to rely on Metro to pick out the right platform.